src/Shared/UnitTests/Vacancies.Client/Infrastructure/ReferenceData/ApprenticeshipProgrammes/ApprenticeshipProgrammesUpdaterServiceTests.cs

```diff
@@ -16,7 +15,7 @@
 using Xunit;
 using ApprenticeshipProgrammesReferenceData = Esfa.Recruit.Vacancies.Client.Infrastructure.ReferenceData.ApprenticeshipProgrammes.ApprenticeshipProgrammes;
 
-namespace Esfa.Recruit.UnitTests.Vacancies.Client.Infrastructure.ReferenceData.ApprenticeshipProgrammes
+namespace Esfa.Recruit.Vacancies.Client.UnitTests.Vacancies.Client.Infrastructure.ReferenceData.ApprenticeshipProgrammes
```

Comment:
Can you remove the Vacancies.Client before the UnitTests part, we need to keep these namespaces manageable and short. Having the same two bits multiple times makes this long and difficult.

Reply:
But shouldnt the namespace match the location of the file?

Reply:
No who said that? This whole thing needs revisiiting, the tests in that folder have all had namespace renamed to what I suggested. Please turn it back. We can address the rest next week.

Reply:
done.